### PR TITLE
Username

### PR DESCRIPTION
**Pull request by brian-smith723**
screen shot 2015-02-19 at 8 18 12 am

### CODE REVIEW

**Comment by FBRTMaka** [comment]
The name should probably go next to the user drop down list button, so you don't need to click the user drop down to find out who you are.

**Comment by DanielJMaher** [comment]
I told him to throw it there for now.  But that makes sense.  So instead of just seeing 
screen shot 2015-02-19 at 10 09 37 am

We would see the button and the name on the nav bar.  You can leave the name in the drop down for now though.

**Comment by brian-smith723** [comment]
to the left or right?


screen shot 2015-02-19 at 2 02 39 pm

**Comment by brian-smith723** [comment]
how's this?

screen shot 2015-02-19 at 2 15 11 pm

**Comment by DanielJMaher** [comment]
Looks fine to me

File: ooiui/static/js/views/common/TOCView.js [hunk 1]
```diff
@@ -90,8 +90,14 @@ var ArrayItemView = Backbone.View.extend({
   },
   onClick: function(e) {
     var self = this;
+    var target = $(e.target)
+    target = target.find('a')
+
+    console.log(target)
```
Comment (birdage) [inline ooiui/static/js/views/common/TOCView.js]: @brian-smith723 can you remove the console log pls

File: ooiui/static/js/views/common/TOCView.js [hunk 2]
```diff
@@ -142,8 +152,12 @@ var PlatformDeploymentItemView = Backbone.View.extend({
   },
   onClick: function(e) {
     var self = this;
+    console.log(e)
```
Comment (birdage) [inline ooiui/static/js/views/common/TOCView.js]: @brian-smith723 can you remove the console log pls

**Comment by birdage** [comment]
@DanielJMaher there we still some changes.....oh well ill fix them

**Pull request merged**